### PR TITLE
TV F7 Migration + PCB bring up

### PR DESCRIPTION
**Pull request by AdityaAsGithub**

### CODE REVIEW

File: source/torque_vector/main.c [hunk 1]
```diff
@@ -36,39 +34,41 @@ GPIOInitConfig_t gpio_config[] = {
...
+    .wake_addr = false,
+    .usart_active_num = USART4_ACTIVE_IDX,
+    .tx_errors = 0,
+    .rx_errors = 0,
     .tx_dma_cfg = &usart_gps_tx_dma_config,
-    .rx_dma_cfg = &usart_gps_rx_dma_config};
+    .rx_dma_cfg = &usart_gps_rx_dma_config
+};
 
 #define TargetCoreClockrateHz 16000000
```
Comment (LukeOxley) [inline source/torque_vector/main.c:73]: will eventually have to check that bumping up the clock rate doesn't break anything

File: source/torque_vector/main.c [hunk 2]
```diff
@@ -94,16 +94,12 @@ SPI_InitConfig_t spi_config = {
     .nss_sw = true,
     .nss_gpio_port = SPI_CS_MAG_GPIO_Port,
```
Comment (LukeOxley) [inline source/torque_vector/main.c:95]: if we are getting rid of bmm stuff, should this port be one of the accel or gps ports?

File: source/torque_vector/main.h [hunk 3]
```diff
@@ -44,6 +44,9 @@
 #define GPS_TX_GPIO_Port (GPIOC)
 #define GPS_TX_Pin (4)
 
+#define GPS_RESET_GPIO_Port (GPIOC)
```
Comment (LukeOxley) [inline source/torque_vector/main.h:47]: lol

**Review by LukeOxley — APPROVED** [review]
Read through comments before merging plz

File: source/torque_vector/imu/imu.c [hunk 4]
```diff
@@ -55,10 +55,6 @@ void imu_periodic(IMU_Handle_t *imu_h, ExtU *rtU)
     r = (int16_t)(imu_h->output.orientation.roll * 10.0f / DEG_TO_RAD);
     y = (int16_t)(imu_h->output.orientation.yaw * 10.0f / DEG_TO_RAD);
 
-    // SEND_ANGLE_DATA(q_tx_can, p, r, y);
-    // SEND_ACCEL_DATA(q_tx_can, (int16_t) (accel_in.x * 100), (int16_t) (accel_in.y * 100), (int16_t) (accel_in.z * 100));
-    // SEND_GYRO_DATA(q_tx_can, (int16_t) (gyro_in.x * 100), (int16_t) (gyro_in.y * 100), (int16_t) (gyro_in.z * 100));
```
[the remark this answers is not in the document]
Comment (AdityaAsGithub) [reply source/torque_vector/imu/imu.c:60]: These messages were never uncommented. The ACCEL and GYRO message already get sent, but if you think we should add the angle message we can.

File: source/torque_vector/bmm/bmm150.c
[the remark this answers is not in the document]
Comment (AdityaAsGithub) [reply source/torque_vector/bmm/bmm150.c:1]: Yes, his enclosure is conductive so there is no way he would be able to get any sort of accurate data which is why I chose to delete it.

**Pull request merged**